packages/medusa/src/services/__tests__/event-bus.ts

```diff
@@ -0,0 +1,152 @@
+import { asClass, asValue, createContainer } from "awilix"
+import config from "../../loaders/config"
+import EventBusService from "../event-bus"
+import { defaultEventBusContainerMock } from "../__fixtures__/event-bus"
+
+jest.genMockFromModule("bull")
+jest.mock("bull")
+jest.mock("../../loaders/config")
+
+// @ts-ignore
+config.redisURI = "testhost"
```

Comment:
suggestion: Not sure this is used in reality no? in this case we can remove otherwise let me know